ui/src/config/section/offering.js

```diff
@@ -89,15 +90,33 @@ export default {
         dataView: true,
         popup: true,
         component: shallowRef(defineAsyncComponent(() => import('@/views/offering/UpdateOfferingAccess.vue')))
+      },
+      {
+        api: 'updateServiceOffering',
+        icon: 'play-circle-outlined',
+        label: 'label.action.enable.service.offering',
+        message: 'message.action.enable.service.offering',
+        dataView: true,
+        args: ['state'],
+        mapping: {
+          state: {
+            value: (record) => { return 'Active' }
+          }
+        },
+        groupAction: true,
+        popup: true,
+        show: (record) => { return record.state !== 'Active' },
+        groupMap: (selection) => { return selection.map(x => { return { id: x, state: 'Active' } }) }
       }, {
         api: 'deleteServiceOffering',
-        icon: 'delete-outlined',
-        label: 'label.action.delete.service.offering',
-        message: 'message.action.delete.service.offering',
+        icon: 'pause-circle-outlined',
```

Comment:
why would you want to change the icon for delete, @vishesh92 ?

Reply:
Because it's not actually deleting the service offering. Just changing the state to inactive.